deployments/nomad/README.md

```diff
@@ -0,0 +1,76 @@
+# Splunk OpenTelemetry Collector for HashiCorp Nomad
+
+The Splunk OpenTelemetry Collector for HashiCorp Nomad is an orchestrator deployment to create a job which provides a unified way to receive, process and export metric, and trace data for [Splunk Observability Cloud](https://www.observability.splunk.com/).
+
+**NOTE**: Job files are provided as a reference only and are not intended for production use._
```

Comment:
Remove the trailing underscore `_` or add a leading one.

Reply:
Added